README.md

```diff
@@ -10,47 +10,63 @@ With the introduction of Visual Studio 2017, Microsoft added some optimizations
 This project converts an existing csproj to the new format, shortening the project file and using all the nice new features that are part of Visual Studio 2017.
 
 ## What does it fix?
-There are a number of things [that VS2017 handles differently](http://www.natemcmaster.com/blog/2017/03/09/vs2015-to-vs2017-upgrade/) that are performed by this tool: 
-1. Include files using a wildcard as opposed to specifying every single file 
-2. A more succinct way of defining project references 
+There are a number of things [that VS2017 handles differently](http://www.natemcmaster.com/blog/2017/03/09/vs2015-to-vs2017-upgrade/) that are performed by this tool:
+1. Include files using a wildcard as opposed to specifying every single file
+2. A more succinct way of defining project references
 3. A more succinct way of handling NuGet package references
 4. Moving some of the attributes that used to be defined in AssemblyInfo.cs into the project file
 5. Defining the NuGet package definition as part of the project file
 
-## How it works
-### As a Net Core Global Tool
-Assuming you have net core 2.1 installed you can run this on the command line:
-`dotnet tool install --global Project2015To2017.Migrate2017.Tool`
-This will install the tool for you to use it anywhere you would like. You can then call the tool as shown in the examples below.
+## Quick Start
+Assuming you have .NET Core 2.1+ installed you can run this on the command line:
+```
+> dotnet tool install --global Project2015To2017.Migrate2017.Tool
+```
 
-### As a normal file download
-Using the tool is simple, it is a simple command line utility that has a single argument being the project file, solution file or folder you would like to convert.
-When you give it a directory path, the tool will discover all csproj files nested in it.
+This will install the tool for you to use it anywhere you would like.
 
-### Examples
-Below examples are for the global tool, for the normal file just replace `dotnet migrate-2017 migrate` with your executable.
+Below examples are for the global tool, for the normal file just replace `dotnet migrate-2017` with your executable.
```

Comment:
If you're removing the line about installing as a normal file, then the latter part of this sentence should be gone